src/components/ScreenStackHeaderConfig.tsx

```diff
@@ -90,7 +87,7 @@ export const ScreenStackHeaderCenterView = (
 };
 
 export const ScreenStackHeaderSearchBarView = (
-  props: React.PropsWithChildren<SearchBarProps>,
+  props: React.PropsWithChildren<ViewProps>,
```

Comment:
This view never supported `SearchBarProps`. No idea why it was typed this way. 

`SearchBar` component does that.